packages/react-native/Libraries/Utilities/Platform.android.js

```diff
@@ -20,6 +20,12 @@ export type PlatformSelectSpec<T> = {
 const Platform = {
   __constants: null,
   OS: 'android',
+  isAndroid: true,
+  isIOS: false,
+  isMacOS: false,
+  isNative: true,
+  isWeb: false,
+  isWindows: false,
```

Comment:
This would be awesome!
But, this is kind of **"redundant"** since:
1. Android and iOS will use and read from their respective files.
2. If the platform is Android this would implicitly mean it's not iOS and vice-versa?
3. Why would an Android app would need a check for MacOS, native, web, and Win similarly for iOS app?
3. If we want to support (check) all these platforms then we should create another _generic file_ (interface) that will define/expose them.

WDYT?
P.S.: I may be unaware of the full context here :)

Reply:
It _is_ certainly redundant. I've just noticed that many RN projects have the same `const isAndroid = Platform.OS === 'android'` declared and thought there way as well be a utility for devs to express one off platform checks this way offered directly by RN.